src/composing/platt_scaling.rs

```diff
@@ -360,7 +367,7 @@ mod tests {
         let prob_values = Array1::linspace(
             F::one() / F::from(n).unwrap(),
             F::one() - F::one() / F::from(n).unwrap(),
-            n - 2,
+            n,
```

Comment:
for evenly spaced values this has to be `n-2` with the given bounds, you have to update the call to `102`

Reply:
Right :D